packages/frontend/app/models/action.ts

```diff
@@ -0,0 +1,27 @@
+import Model, { attr } from '@ember-data/model';
+
+import VoteChain from 'emberclear/models/vote-chain';
+
+export enum ACTION_RESPONSE {
+  NONE = 'none',
+  APPROVED = 'approved',
+  DENIED = 'denied',
+  DISMISSED = 'dismissed',
+}
+
+export default class Action extends Model {
+  @attr() vote!: VoteChain;
```

Comment:
This should likely be a Vote rather than a VoteChain. Vote stays constant while its VoteChain is updated in the background. Also relationships with other models should use belongsTo rather than attr. See the other channel models to see some examples.